docs/features/dns-cache.rst

```diff
@@ -0,0 +1,27 @@
+User experience may be greatly improved when dns is accelerated. Also it
+seems like a good idea to keep the number if packages being exchanged
+between node and gateway as small as possible. In order to do this, a
+dns-cache may be used on a node. The dnsmasq instance listening on port
+53 in the node will be re-configured to answer requests, use a list of
+upstream servers and a specific cache size if the below options are
+added to site.conf All settings are optional, though if no dns server is
+set, the configuration will not be altered by gluon-core.
+
+Besides caching dns requests from clients, the next_node-addresses are set to
+resolve to a configurable name that may optionally be placed in next_node.name.
+.. literal:: 
+ 
+ dns = {
+      cacheentries = 5000,
+      servers = { '2a06:8187:fb00:53::53', },
+ },
+  
+ next_node = {
+      name = 'nextnode',
+      ip6 = 'dddd::1',
+      ip4 = '1.2.3.4',
+ }
```

Comment:
All addresses in this section should use the designated documentation prefixes (192.0.2.0/24, 198.51.100.0/24 and 203.0.113.0/24 [[RFC5737](https://tools.ietf.org/html/rfc5737)]; 2001:db8::/32 [[RFC3849](https://tools.ietf.org/html/rfc3849)]).